### ISSUE TITLE
Generate static site

### ISSUE THREAD

**Issue by danhoran** [webpackmonitor/webpackmonitor#63]
Firstly, I just want to say what a fantastic tool this is.

I'm wondering whether it's possible to generate a static site from the results, rather than the server-based solution? If it's not currently possible, is it on your roadmap anywhere?

**Comment by neutraali** [comment]
Yeah, shouldn't be too much of an issue to switch from `dev-server` to `build` - +1 for this.

**Comment by joonarahko** [comment]
@danhoran we need this too : D
